recipes/affinder/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "affinder" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/affinder-{{ version }}.tar.gz
+  sha256: 77ce5a4602c50fb60e1ad4c52c2ec4a865e80f1a7057af061c06ca3bc8b0e666
+
+build:
+  entry_points:
+    - affinder=affinder.main:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - magicgui >=0.3.3
+    - napari >=0.4.12
+    - napari-plugin-engine >=0.1.9
+    - numpy
+    - python >=3.7
+    - scikit-image
+    - toolz
+
+test:
+  imports:
+    - affinder
+  commands:
+    - pip list
+    - pip check
+    - export QT_DEBUG_PLUGINS=1  # [not win]
+    - affinder --help  # [not linux]
```

Is this selector due to the virtual framebuffer issues? Maybe this helps:

```suggestion
    - affinder --help # [not linux]
    - DISPLAY=localhost:1.0 xvfb-run -a affinder --help  # [linux]
```